cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/namespace/DefaultNamespaceAdmin.java

```diff
@@ -76,9 +75,9 @@ public DefaultNamespaceAdmin(StoreFactory storeFactory, PreferencesStore prefere
    */
   private void createDefaultNamespace() {
     NamespaceMeta.Builder builder = new NamespaceMeta.Builder();
-    NamespaceMeta defaultNamespace = builder.setId(Constants.DEFAULT_NAMESPACE)
-      .setName("Default namespace")
-      .setDescription("The default namespace, a reserved system namespace.")
+    NamespaceMeta defaultNamespace = builder
+      .setName(Constants.DEFAULT_NAMESPACE)
+      .setDescription("Default Namespace")
```

Comment:
I see your point. Maybe we can call it that? However, we don't really expose the concept of user vs system namespace yet. Maybe best to keep it as "Default Namespace" then to avoid creating any confusion.